CMakeLists.txt

```diff
@@ -182,11 +182,8 @@ if(GINKGO_BUILD_HWLOC AND (MSVC OR WIN32 OR CYGWIN OR APPLE))
     message(WARNING "Ginkgo does not support HWLOC on Windows/MacOS, switch GINKGO_BUILD_HWLOC to OFF")
 endif()
 if(GINKGO_BUILD_HWLOC)
-    # By default always use external HWLOC
-    set(GINKGO_USE_EXTERNAL_HWLOC 1)
     set(GINKGO_HAVE_HWLOC 1)
```

Comment:
can this variable `GINKGO_HAVE_HWLOC` also be deleted? `GINKGO_HAVE_HWLOC` seems to be equal to `GINKGO_BUILD_HWLOC`

Reply:
We use this to have a `define` which is set to `0` or `1`. If we use the `BUILD` instead we also need to change all macros to use `#ifdef GKO_HAVE_HWLOC` rather than `#if GKO_HAVE_HWLOC`.